site/_posts/2018-01-30-manageiq-gaprindashvili-ga-announcement.html.md

```diff
@@ -0,0 +1,67 @@
+
+You can find this new tool under Automation -> Automate -> Generic Objects.
+
+[![Generic Object Class](/assets/images/blog/Gaprindashvili_GA_generic_objects.png)](/assets/images/blog/Gaprindashvili_GA_generic_objects.png)
+
+## Lenovo XClarity Provider
+
+We introduced the Physical Infrastructure provider in our [last release](/blog/2017/05/manageiq-fine-ga-announcement/). In this release, the first implementation is [Lenovo XClarity Administrator](https://github.com/ManageIQ/manageiq-providers-lenovo). This avails monitoring, control, and alerting for physical infrastructures that provide the foundation for on-premise clouds.
+
+With this integration, ManageIQ gives a unified interface for users to obtain information regarding inventory, monitoring, roles, policies and reporting across their entire data center infrastructure.
+
+[![Lenovo Provider](/assets/images/blog/Gaprindashvili_GA_Lenovo_MSV.png)](/assets/images/blog/Gaprindashvili_GA_Lenovo_MSV.png)
+
+## Provider Improvements
+
+With some major work in the [AmazonSsaSupport repo](https://github.com/ManageIQ/amazon_ssa_support), you can now run Smart State Analysis on Amazon Instances and Images. As a result, Compliance and Policy Enforcement can be applied to Amazon instances. In addition, graph refresh and targeted refresh are enabled by default, providing performance improvement.
+
+[![Amazon Refresh](/assets/images/blog/Gaprindashvili_GA_Amazon_refresh.png)](/assets/images/blog/Gaprindashvili_GA_Amazon_refresh.png)
+
+Smart State Analysis is available on Microsoft Azure Managed Disks as well.
+
+On the container front, you can now provision with OpenShift Templates from the Service Catalog. There are new metering reports for container images and projects, as well as support for Chargeback by allocation. Coming full circle, ManageIQ can be deployed on the OpenShift Container Platform!
+
+In oVirt, we have targeted refresh improvements for templates and host events. The OVN (Open Virtual Network) has been added as oVirt's network provider for network switch visualization and control.
+
+And as a shift away from real time metrics for alerts, ManageIQ can now receive push alerts from Prometheus in OpenShift. The Prometheus Alerts Collection Manager is added to Container Providers, and visible through the Monitor -> Alerts menu.
+
+[![Prometheus Alerts](/assets/images/blog/Gaprindashvili_GA_Prometheus_alerts.png)](/assets/images/blog/Gaprindashvili_GA_Prometheus_alerts.png)
+
+## All Good Things...
+
+We are deprecating the MiqLdap client in this release, which provides support for the Authentication Mode: LDAP and LDAPS. An automated conversion script, call `miqldap_to_sssd`, is provided to convert ManageIQ Authentication configurations that use the MiqLdap client to using External authentication with SSSD. See this [blog post](/blog/2017/09/miqldap-to-sssd/) describing how to use the `miqldap_to_sssd` conversion tool.
```

Comment:
> conversion script, call `miqldap_to_sssd`

The word "call" should be "called"

Reply:
Looks good.

Just one nit:

I suggest rewording:

An automated conversion script, call `miqldap_to_sssd`, is provided to convert ManageIQ Authentication configurations that use the MiqLdap client to using External authentication with SSSD.

To be:

An automated conversion script, call `miqldap_to_sssd`, is provided that will  convert ManageIQ Authentication configurations from using the MiqLdap client into Authentication configurations that will use External authentication with SSSD.